### ISSUE TITLE
homie-control installed, links not working - 404 not found

### ISSUE THREAD

**Issue by 466** [stufisher/homie-control#2]
I installed nginx with php, which does seem to work atm.
homie-control is reachable via homie.local from my LAN.

I can see the main page of homie-control, without any devices being activated.
So i can see and click on:
History, Properties, Devices, Groups, Schedules, Profiles, Mobile Devices, Triggers, Options and Repeater.

Sadly, that just leads to:
e.g., history:
http://homie.local/history
404 Not Found

Any ideas what i did wrong?
Is there a working nginx config out there?

Thanks and BR!

**Comment by stufisher** [comment]
You'll need to translate the `api/.htaccess` and `client/.htaccess` to nginx rewrite rules

I would usually setup
DocumentRoot = client
move api into client folder

Sorry i should add this to the install instructions!

**Comment by 466** [comment]
Yeha, thanks for that hint, translating the rewrite rules did it, at least most of it :)
I used this online converter: https://timmehosting.de/htaccess-converter
which converted client/.htaccess into:
```
try_files $uri $uri/ /index.php$is_args$args;
```
so i added this to my nginx config:
```
location /
{
try_files $uri $uri/ /index.php$is_args$args;
}
```
using the setup you suggested.

So i can click the links and see the new pages, but on those, clicking any button, does not do anything ... .
e. g. 
profile, add profile / add component
trigger, add property trigger / add sun trigger
schedule, add schedule

what actually does work is:
groups, add group

As you said, there are two htaccess files, client/ and client/api, perhaps this behavior is due to me only using the translated client/.htaccess ?

**Comment by stufisher** [comment]
What happens when you load 
localhost/api ?

**Comment by 466** [comment]
localhost/api 
returns:

> Page Not Found
> 
> The page you are looking for could not be found. Check the address bar to ensure your URL is spelled correctly. If all else fails, you can visit our home page at the link below.
> Visit the Home Page

**Comment by 466** [comment]
Some place special, i can see, what happens / what does not happen, if one clicks any of those buttons?

**Comment by stufisher** [comment]
Yes in chrome ctrl-alt-i brings up the developer console, then network tab to see what requests are being made. Sounds like your api is not configured either
